.appveyor.yml

```diff
@@ -101,20 +96,12 @@ install:
 
           # download Scrutinizer's Ocular
           if (!(Test-Path C:\tools\ocular)) {
-            New-Item -path c:\tools -name ocular -itemtype directory
+            New-Item -Path c:\tools -Name ocular -ItemType Directory | Out-Null
           }
           if (!(Test-Path c:\tools\ocular\ocular.phar)) {
             appveyor-retry appveyor DownloadFile https://github.com/scrutinizer-ci/ocular/releases/download/1.5.2/ocular.phar -Filename C:\tools\ocular\ocular.phar
             Set-Content -path 'C:\tools\ocular\ocular.bat' -Value ('@php C:\tools\ocular\ocular.phar %*')
           }
-
-          # download CA bundle
-          if (!(Test-Path C:\tools\cacert)) {
-            New-Item -path c:\tools\ -name cacert -itemtype directory
-          }
-          if (!(Test-Path c:\tools\cacert\bundle.pem)) {
-            appveyor-retry appveyor DownloadFile https://curl.haxx.se/ca/cacert.pem -Filename C:\tools\cacert\bundle.pem
-          }
         }
```

Comment:
Just noticed now, PHP block should end before installing Composer/Ocular - could you please move it above `download Composer`? Thanks.

Reply:
What about `appveyor-retry cinst -y sqlite`?

Reply:
That should stay as part of the PHP block.
The question is whether we even need it, we don't test against SQLite (although we originally wanted to). Imho we can drop SQLite completely.